openfisca_extension_template/parameters/local_town/child_allowance/index.yaml

```diff
@@ -1 +1,2 @@
+---
```

Comment:
Why do you need this `---` ?

Reply:
It is just yamllint that asks for it. I didn't give it a thought.

Reply:
This was the weird thing ;-)